src/Controller/QuestionController.php

```diff
@@ -53,24 +60,55 @@ public function homepage(QuestionRepository $repository, int $page = 1)
     }
 
     /**
```

Comment:
Change this annotation out to the new style and use the Security method for the is_granted check

Reply:
Done